deps/argon2/argon2/blake2/blamka-round-opt.h

```diff
@@ -20,12 +20,17 @@
 
 #include "blake2-impl.h"
 
+#if defined(__x86_64__) || defined(_M_X64) || defined(_M_AMD64)
 #include <emmintrin.h>
+#elif defined(__aarch64__) || defined(_M_ARM64)
+#include "sse2neon.h"
+#endif
+
 #if defined(__SSSE3__)
 #include <tmmintrin.h> /* for _mm_shuffle_epi8 and _mm_alignr_epi8 */
 #endif
 
-#if defined(__XOP__) && (defined(__GNUC__) || defined(__clang__))
+#if (defined(__x86_64__) || defined(_M_X64) || defined(_M_AMD64)) && defined(__XOP__) && (defined(__GNUC__) || defined(__clang__))
```

Comment:
Actually, why do you check the windows variable here?

Reply:
not only the windows. it check wether in x86_64 or not. linux will also hit this condition.

Reply:
yes but _M_X64 and _M_AMD64 are for MSVC and not linux, no? So why would you need to check for them here when we dont previously do that

Reply:
Yes, you are right. x86intrin.h is for gcc/clang. MSVC uses intrin.h.
I will fix this. only __x86_64__ need to be check.